packages/manager-components/src/utils/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,10 @@
+{
+  "price_ht_label": "HT",
+  "price_ttc_label": "TTC",
+  "price_free": "Inclus",
+  "price_gst_excl_label": "ex. GST",
+  "price_gst_incl_label": "incl. GST",
+  "price_per_month": "/mois ",
+  "price_per_year": "/an ",
+  "price_per_day": "/jour "
```

Comment:
Extra spaces for presentation should be handled by the html / css, not the translation